website/react-magma-docs/src/pages/api/date-pickers.mdx

```diff
@@ -95,6 +95,48 @@ export function Example() {
 }
 ```
 
+## Clearing the Date
+
+After a date has been selected the user can programmatically clear the input field and the `Chosen Date:` field via `isClearable` X button or `Clear Date` button.
```

Comment:
We can make this more generic since we don't know the exact use case for date clearing. I think it would be enough to say something along the lines of ```Dates can be cleared with the `isClearable` prop, or programmatically through the `onDateChange` function by passing null.```

Reply:
Updated. Thank you for the idea. So far it's difficult for me to come up with a description